bitnami/minio/templates/_helpers.tpl

```diff
@@ -275,6 +275,13 @@ minio: persistence.accessModes
 {{- end -}}
 {{- end -}}
 
+{{/*
+Label used to uniquely identify MinIO&reg; gateway deployment
+*/}}
+{{- define "minio.gateway.pdb.selector" -}}
+app: minio.gateway
+{{- end -}}
+
```

Comment:
I'd remove this helper since `nameOverride: "minio.gateway"` would achieve the same here right? Or am I missing something?

Reply:
Good point, I had a misunderstanding in how PDB works exactly. Your suggestions applied in d6ac67c